packages/mobile/src/web3/saga.ts

```diff
@@ -48,30 +45,24 @@ let AssignAccountLock = false
 
 // checks if web3 claims it is currently syncing or not
 function* checkWeb3SyncProgressClaim() {
-  Logger.debug(TAG, 'Checking sync progress claim')
   while (true) {
     try {
-      const syncProgress = yield web3.eth.isSyncing()
-      Logger.debug(TAG, 'Sync progress', syncProgress)
-
+      const syncProgress = yield web3.eth.isSyncing() // returns true when it's still syncing and thus not ready
       if (typeof syncProgress === 'boolean' && !syncProgress) {
-        // For some weird reason, checkSyncProgressWorker is flaky and does not work for the long running
-        // sync tasks.
-        Logger.debug(TAG, 'sync complete')
+        Logger.debug(TAG, 'checkWeb3SyncProgressClaim', 'sync complete')
+
         yield put(setSyncProgress(100))
```

Comment:
Does anything still consume syncProgress?

Reply:
Currently, no. I feel that code might be useful in the future.